src/authentication.rs

```diff
@@ -83,7 +103,7 @@ impl AuthUser {
 
     pub async fn from_session(session: Session, tx: &mut AppTx) -> ResponseResult<Self> {
         let user_id: Uuid = session
-            .get("auth_user_id")
```

Comment:
Nice catch!

Reply:
I've updated the code, please review it again